.github/workflows/linux_meson.yml

```diff
@@ -59,12 +59,12 @@ jobs:
         sudo apt-get install -y libopenblas-dev libatlas-base-dev liblapack-dev gfortran libgmp-dev libmpfr-dev libsuitesparse-dev ccache libmpc-dev
 
     - name: Install Python packages
-      if: matrix.python-version == '3.9'
+      if: matrix.python-version == '3.10'
       run: |
         python -m pip install numpy cython pytest pytest-xdist pytest-timeout pybind11 mpmath gmpy2 pythran ninja meson click rich-click doit pydevtool pooch
 
     - name: Install Python packages from repositories
-      if: matrix.python-version == '3.11-dev'
+      if: matrix.python-version == '3.12-dev'
```

Comment:
This is a few months too early probably - normally we start around May-June. I'd make this `3.11` and add a comment to switch it to `3.12-dev` in 4 months